tools/check_format.py

```diff
@@ -94,15 +93,20 @@ def checkFormatVisitor(arg, dir_name, names):
 
 
 if __name__ == "__main__":
-  if len(sys.argv) != 2 and len(sys.argv) != 3:
-    print(USAGE)
-    sys.exit(1)
-
-  operation_type = sys.argv[1]
-  target_path = sys.argv[2] if len(sys.argv) == 3 else "."
+  parser = argparse.ArgumentParser(description='Check or fix file format')
+  parser.add_argument('operation_type', type=str, help="specify if the run should 'check' or 'fix' format.")
+  parser.add_argument('target_path', type=str, nargs="?", default=".", help="specify the root directory"
+                                                                            "for the script to recurse over. Default '.'.")
+  parser.add_argument('--add_excluded_prefixes', type=str, nargs="+", help="exclude additional prefixes.")
+  args = parser.parse_args()
+
+  operation_type = args.operation_type
+  target_path = args.target_path
+  if args.add_excluded_prefixes:
+    EXCLUDED_PREFIXES += tuple(args.add_excluded_prefixes)
```

Comment:
As a nit, I think it should be a `list` since it's now mutable, but it barely matters here.